devfiles/java-mysql/devfile.yaml

```diff
@@ -24,7 +25,13 @@ components:
           -Duser.home=/home/user"
       - name: MAVEN_OPTS
         value: $(JAVA_OPTS)
+<<<<<<< Updated upstream
```

Comment:
is this line expected to be here?

Reply:
that's weird. looks as if VisualStudioCode didn't rewrite the file
I'll fix it